vpr/src/pack/pack.h

```diff
@@ -17,7 +18,8 @@ bool try_pack(t_packer_opts* packer_opts,
               const t_model* user_models,
```

Comment:
It would be good to write a doxygen comment for this routine, documenting at least the flat_placement_info parameter (and more if you know what they are for).